src/steps/sensors/converters.ts

```diff
@@ -16,6 +16,8 @@ export function createSensorEntity(data: Sensor) {
         _key: getSensorKey(data.guid),
         name: data.guid,
         category: ['sensor'],
+        // TODO: can you map all the fields in here?
+        // Similar to malops/converter.ts (just you don't have use the custom map function)
```

Comment:
Sure. Just for the sensors, or for all the remaining entities as well?

Reply:
For the remaining entities as well - we want to map over most/if not all fields inside of our converters :slightly_smiling_face: